src/cswinrt/code_writers.h

```diff
@@ -1936,8 +1939,15 @@ MarshalInspectable<object>.DisposeAbi(ptr);
                         has_base_type ? ":base(global::WinRT.DerivedComposed.Instance)" : "",
                         cache_object,
                         method.Name(),
-                        bind_list<write_parameter_name_with_modifier>(", ", signature.params())
-                    );
+                        bind_list<write_parameter_name_with_modifier>(", ", signature.params()),
+                        bind([&](writer& w)
+                        {
+                            if (is_manually_gen_default_interface)
+                            {
+                                auto default_interface_name_non_ABI = get_default_interface_name(w, class_type, false);
```

Comment:
consider projected_default_interfance_name as projected is the naming convention used for non_abi.